alibi_detect/od/pytorch/knn.py

```diff
@@ -0,0 +1,102 @@
+from typing import Optional, Union, List, Tuple
+
+import numpy as np
+import torch
+
+from alibi_detect.od.pytorch.ensemble import Ensembler
+from alibi_detect.od.pytorch.base import TorchOutlierDetector
+
+
+class KNNTorch(TorchOutlierDetector):
+    def __init__(
+            self,
+            k: Union[np.ndarray, List, Tuple],
+            kernel: Optional[torch.nn.Module] = None,
+            ensembler: Optional[Ensembler] = None,
+            device: Optional[Union[str, torch.device]] = None
+            ):
+        """PyTorch backend for KNN detector.
+
+        Parameters
+        ----------
+        k
+            Number of neirest neighbors to compute distance to. `k` can be a single value or
```

Comment:
Type hint doesn't allow for single values here, also not clear that these should be `int`.